frontends/bsd/src/app_root.tsx

```diff
@@ -566,6 +576,20 @@ export function AppRoot({
     );
   }
 
+  if (smartcard.status !== 'no_card' && !cardRemovedAfterAuth) {
```

Comment:
The new `cardRemovedAfterAuth` variable name has made everything much clearer - nice!